chord_metadata_service/chord/views_search.py

```diff
@@ -305,6 +308,20 @@ def mcodepacket_query_results(query, params, options=None):
     return queryset
 
 
+def get_biosamples_experiments(subject_ids):
+    biosamples = Biosample.objects.filter(phenopacket__subject_id__in=subject_ids)\
+        .values(
+            subject_id=F("phenopacket__subject_id"),
+            biosample_id=F("id"),
+            experiment_id=F("experiment__id"),
```

Comment:
there can be more than one experiment on a biosample, so what happens in that case?

Reply:
the function fetches biosample objects that are associated with the list of subject IDs, and it returns a list of dictionaries the represents a relation between a biosample and the experiment. It includes all associated experiments in the results.

Reply:
can you add a comment saying that this returns a queryset which will give an entry for each experiment that has an associated biosample or something like that? and maybe rename the function to match what it gets assigned to below

Reply:
Added! Thanks for the suggestion!